decidim-core/app/cells/decidim/content_blocks/static_page/section/show.erb

```diff
@@ -0,0 +1 @@
+<%= content %>
```

Comment:
there is no other format needed here ?

Reply:
Nope. Just like with the summary content block, we need to show the content on the page.